### PR TITLE
Model Compiler - Update & Bug Fixes

### PR DESCRIPTION
**Pull request by kiritigowda**
## Model Compiler 
* include dirs updated
* rocm path added
* Fix bugs

### CODE REVIEW

File: model_compiler/python/nnir_to_clib.py [hunk 1]
```diff
@@ -298,8 +298,8 @@ def generateCMakeExtras(graph, outputFolder):
 endif()
 
 find_package(OpenCV QUIET)
-include_directories (/opt/rocm/include/miviisionx ../)
-link_directories    (/opt/rocm/lib)
+include_directories (${ROCM_PATH}/include ${ROCM_PATH}/include/mivisionx ../)
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_clib.py:301]: @hansely can you confirm why we have `../` as one of the dirs? is this typo or can we fix this with a full link?

Comment (hansely) [reply model_compiler/python/nnir_to_clib.py:301]: @kiritigowda sorry that is a typo. Would you be able to remove it in this PR?

**Comment by paveltc** [comment]
@kiritigowda What are the relevant tests for this PR?

**Comment by kiritigowda** [comment]
@paveltc - This PR would need to run NN tests and mv_compile tests to validate. @hansely add any tests you think we need apart for those?

**Comment by hansely** [comment]
@paveltc @kiritigowda The only related app to nnir_to_clib.py script is mv_compile so far. Therefore, we won't need any extra unit test for this PR.

**Review by hansely — APPROVED** [review]

**Pull request merged**